lib/safeRequire.js

```diff
@@ -3,7 +3,9 @@ module.exports = function safeRequire (library, feature) {
 		return require(library);
 	} catch (error) {
 		if (error.code === 'MODULE_NOT_FOUND') {
-			throw new Error('\nTo use ' + feature + ' install ' + library);
+			console.error('\nTo use ' + feature + ' install ' + library);
+			process.exit(1);
+			return;
```

Comment:
ah, the always return even null. a null return after an IO call seems even weirder from a type system perspective :stuck_out_tongue: